readme.md

```diff
@@ -139,33 +133,10 @@ Limitations:
 
 # Troubleshooting
 
-- If you have fsevents trouble, first nuke your node_modules
-
-## Running on Catalina with Nix
-
-Things seem to be changing every day but for now, our solution is taken from here:
-https://github.com/NixOS/nix/issues/2925#issuecomment-539490866
-
-```
-sudo mkdir /System/Volumes/Data/opt/nix
-sudo chown {your_user} /System/Volumes/Data/opt/nix
-# Be careful as the space needs to be a tab, otherwise changes won't be picked up from synthetic.conf
-sudo sh -c "echo 'nix	System/Volumes/Data/opt/nix' >> /System/Volumes/Data/private/etc/synthetic.conf"
-```
-
-^ make sure you replace the spaces with a tab character as directed above **or this will NOT work**.
-
-Reboot, and then:
-
-```
-curl -L https://nixos.org/nix/install | sh
-```
-
-Add the `. /Users/{{USERNAME}}/.nix-profile/etc/profile.d/nix.sh` line to your shell's config file (e.g. ~/.zshrc, ~/.bashrc) as directed by the end of the nix install.
-
-And while you're at it, this as well, which will supress the error that you're using a symlink instead of a real directory: `export NIX_IGNORE_SYMLINK_STORE=1`
+## fsevents
+If you have fsevents trouble, try nuking your node_modules and reinstalling with `npm install`
```

Comment:
```suggestion
If you notice that 1 or more CPU cores are running 100% because of `node` processes, it is probably webpack-dev-server having trouble with `fsevents` on MacOS. To fix it, run `npm install fsevents` in the `utopia/editor` directory. see https://github.com/webpack/webpack/issues/701#issuecomment-216082726
```